tensorflow/python/grappler/hierarchical_controller.py

```diff
@@ -464,6 +464,8 @@ def create_op_embeddings(self, verbose=False):
 
     self.type_dict = {}
     for op_type in self.cluster.ListAvailableOps():
+      if isinstance(op_type, bytes):
```

Comment:
Is this still needed ? cluster.ListAvailableOps() should now return strings and not bytes.

Reply:
I was looking at the latest nightly build and it appeared to be returning bytes.  I haven't built from source to see though.